optaplanner-docs/src/main/asciidoc/Chapter-Planner_introduction/Chapter-Planner_introduction.adoc

```diff
@@ -11,48 +11,44 @@
 == What is OptaPlanner?
 
-*https://www.optaplanner.org[OptaPlanner] is a lightweight, embeddable constraint satisfaction engine which optimizes planning problems.* It solves use cases such as:
+Every organization faces planning problems: providing products or services with a limited set of _constrained_ resources (employees, assets, time and money). OptaPlanner optimizes such planning to do more business with less resources.
+This is known as _Constraint Satisfaction Programming_ (which is part of the _Operations Research_ discipline).
+
+https://www.optaplanner.org[OptaPlanner] is a lightweight, embeddable constraint satisfaction engine which optimizes planning problems. It solves use cases such as:
 
 * **Employee shift rostering**: timetabling nurses, repairmen, ...
 * **Agenda scheduling**: scheduling meetings, appointments, maintenance jobs, advertisements, ...
 * **Educational timetabling**: scheduling lessons, courses, exams, conference presentations, ...
-* **Vehicle routing**: planning vehicles (trucks, trains, boats, airplanes, ...) with freight and/or people
-* **Bin packing**: filling containers, trucks, ships and storage warehouses, but also cloud computers nodes, ...
+* **Vehicle routing**: planning vehicle routes (trucks, trains, boats, airplanes, ...) for moving freight and/or passengers through multiple destinations using known mapping tools ...
+* **Bin packing**: filling containers, trucks, ships, and storage warehouses with items, but also packing information across computer resources, as in cloud computing ...
 * **Job shop scheduling**: planning car assembly lines, machine queue planning, workforce task planning, ...
 * **Cutting stock**: minimizing waste while cutting paper, steel, carpet, ...
-* **Sport scheduling**: planning football leagues, baseball leagues, ...
+* **Sport scheduling**: planning games and training schedules for football leagues, baseball leagues, ...
 * **Financial optimization**: investment portfolio optimization, risk spreading, ...
 
 image::Chapter-Planner_introduction/useCaseOverview.png[align="center"]
 
-Every organization faces planning problems: provide products or services with a limited set of _constrained_ resources (employees, assets, time and money). OptaPlanner optimizes such planning to do more business with less resources.
-This is known as _Constraint Satisfaction Programming_ (which is part of the _Operations Research_ discipline).
-
-image::Chapter-Planner_introduction/whatIsAPlanningProblem.png[align="center"]
-
-OptaPlanner helps normal Java^TM^
- programmers solve constraint satisfaction problems efficiently.
-Under the hood, it combines optimization heuristics and metaheuristics with very efficient score calculation.
-
-
-[[requirements]]
-== Requirements
+[[whatIsAPlanningProblem]]
+== What is a Planning Problem?
+//image::Chapter-Planner_introduction/whatIsAPlanningProblem.png[align="center"]
+//I propose removing this image and replacing it with the following paragraph.
```

Comment:
I'd prefer to keep the image but keep your text too (for 508). A bit of graphical icons to illustrate what these 3 sentence parts are don't hurt no? We also use this image in our slidedeck.